src/documenti/interni/verbali/2022_02_21/contenuti/todo.tex

```diff
@@ -0,0 +1,10 @@
+\section{Todo}
+Durante la riunione sono emersi i seguenti task da svolgere.
+
+\begin{center}
+  \begin{tabular}{|p{5cm}|p{7cm}|}
+    \hline
+    Tutti & Conclusione della sistemazione dei documenti \\ \hline
```

Comment:
Non so se avrebbe senso scrivere i vari compiti così da rendere questo diario utile in questi due giorni, ma credo che tutti sappiano quindi forse una perdita di tempo

Reply:
Direi che per 2 giorni non serve, penso ognuno sappia cosa debba fare, ho messo giusto quello mio e di Marko perché è una cosa "nuova"

Reply:
sisi penso vada bene cos'